### ISSUE TITLE
Import of templates with labels doesn't work

### ISSUE THREAD

**Issue by sumkincpp** [ansible/tower-cli#625]
tower-cli version: latest upstream
AWX version: 1.0.6.49

Create workflow template using json format(or via `tower-cli receive  --workflow`) with labels attached, i.e. some label should be present in template definition :
```json
...
    "asset_relation": {
      "labels": [
        {
          "organization": "MyOrg",
          "name": "accepted"
        }
      ],
...
```

If template with label isn't present in AWX, sending it will result in TypeError exception
```bash
# tower-cli send test_w.json
WORKFLOW [WORKFLOW | XXXX] **************************************************************************************************
Checking on workflow_nodes
Asset up to date
Added label accepted
Traceback (most recent call last):
  File "/opt/awx/awx-app/venv/bin/tower-cli", line 11, in <module>
    load_entry_point('ansible-tower-cli', 'console_scripts', 'tower-cli')()
  File "/usr/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/opt/awx_builddir/resources/pip/ansible-tower-cli/tower_cli/cli/base.py", line 129, in invoke
    return super(TowerCLI, self).invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/opt/awx_builddir/resources/pip/ansible-tower-cli/tower_cli/conf.py", line 359, in method_with_context_managed
    method(*args, **kwargs)
  File "/opt/awx_builddir/resources/pip/ansible-tower-cli/tower_cli/cli/misc.py", line 370, in send
    sender.send(source, prevent, exclude, secret_management)
  File "/opt/awx_builddir/resources/pip/ansible-tower-cli/tower_cli/cli/transfer/send.py", line 294, in send
    self.import_labels(existing_object, relations[a_relation], asset_type)
  File "/opt/awx_builddir/resources/pip/ansible-tower-cli/tower_cli/cli/transfer/send.py", line 1010, in import_labels
    'label': created_label['id'],
  File "/opt/awx_builddir/resources/pip/ansible-tower-cli/tower_cli/models/fields.py", line 198, in method
    return internal_method(method._relationship, obj_pk, other_obj_pk)
  File "/opt/awx_builddir/resources/pip/ansible-tower-cli/tower_cli/models/base.py", line 569, in _assoc
    url = self.endpoint + '%d/%s/' % (me, url_fragment)
TypeError: %d format: a number is required, not NoneType
```

**Comment by axnoack** [comment]
For Tag 3.3.3 change this line https://github.com/ansible/tower-cli/blob/v3.3.3/tower_cli/cli/transfer/send.py#L967 from

> keyword_arg_template_id = "workflow_job_template"

to

> keyword_arg_template_id = "workflow"

or use the master

**Comment by jainnikhil30** [comment]
@axnoack 

Yeah this is fixed via #683, it will be released with 3.3.4
